src/vorta/application.py

```diff
@@ -46,11 +46,11 @@ def __init__(self, args_raw, single_app=False):
         if self.isRunning():
             if single_app:
                 self.sendMessage("open main window")
-                print('An instance of Vorta is already running. Opening main window.')
+                print('An instance of Vorta is already running. Opening main window.')  # noqa: T20
```

Comment:
Could this be a log statement instead?

Reply:
I don't really know why this was a print statement at the first place, can you give me some context? cc:@real-yfprojects

Reply:
I am to blame for this line.
log would indeed be better